test/built-ins/Iterator/proto-from-ctor-realm.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-iterator
+description: Default [[Prototype]] value derived from realm of the NewTarget.
+info: |
```

Comment:
Bad frontmatter that's preventing the test harness from picking up `features:`. I guess the `|` starts a YAML literal string or some shit and it's causing `features:` to not be parsed correctly. Since info is empty I'd just remove this line.
```suggestion
```